files/plugins/cron_cis_audit.py

```diff
@@ -47,26 +48,36 @@ def _get_major_version():
 
 MAX_SLEEP = 600
 PID_FILENAME = "/tmp/cron_cis_audit.pid"
+TAILORING_CIS_FILE = Path("/etc/usg/default-tailoring.xml")
 
 
 def _get_cis_hardening_profile(profile):
-    """Try to read the cis profile from cloud init log or default to level1_server."""
+    """Try to read the cis profile from cloud init log or default to level1_server.
+
+    If exists the "default" tailoring file in /etc/usg/default-tailoring.xml,
+    no profile is passed.
+    """
     if profile in PROFILES:
-        return profile
+        return [profile]
+
+    if TAILORING_CIS_FILE.exists():
+        return []
 
     if not os.path.exists(CLOUD_INIT_LOG) or not os.access(CLOUD_INIT_LOG, os.R_OK):
         print(
             "{} not existing/accessible, default to profile '{}'".format(
                 CLOUD_INIT_LOG, DEFAULT_PROFILE
             )
         )
-        return DEFAULT_PROFILE
+        return [DEFAULT_PROFILE]
+
     pattern = re.compile(r"Applying Level-(1|2) scored (server|workstation)")
     for _, line in enumerate(open(CLOUD_INIT_LOG)):
         for match in re.finditer(pattern, line):
             level, machine_type = match.groups()
-            return "level{}_{}".format(level, machine_type)
-    return DEFAULT_PROFILE
+            return ["level{}_{}".format(level, machine_type)]
+
+    return [DEFAULT_PROFILE]
```

Comment:
Nit: since this will only ever be a list with 1 or 0 elements, wouldn't it be more readable to return a string or None?